src/ECER.Managers.Registry/ApplicationHandlers.cs

```diff
@@ -4,12 +4,12 @@
 using ECER.Engines.Validation.Applications;
 using ECER.Infrastructure.Common;
 using ECER.Managers.Registry.Contract.Applications;
-using ECER.Resources.Accounts.Registrants;
 using ECER.Resources.Documents.Applications;
 using ECER.Resources.Documents.PortalInvitations;
 using ECER.Utilities.DataverseSdk.Model;
 using MediatR;
 using CharacterReferenceSubmissionRequest = ECER.Managers.Registry.Contract.Applications.CharacterReferenceSubmissionRequest;
+using OptOutReferenceRequest = ECER.Managers.Registry.Contract.Applications.OptOutReferenceRequest;
```

Comment:
I personally prefer to fully qualify the type instead of using 'using x = ' as it is easier on maintenance and code reading